.github/goss_validate_deb.yaml

```diff
@@ -0,0 +1,28 @@
+---
+package:
+  alumet-agent:
+    installed: true
+
+service:
+  alumet:
+    enabled: false
+    running: false
+
+file:
+  /usr/lib/alumet-agent:
+    exists: true
+    mode: "0755"
+  /usr/lib/systemd/system/alumet.service:
+    exists: true
+    mode: "0644"
+  /usr/bin/alumet-agent:
+    exists: true
+    mode: "0755"
+  /etc/alumet/alumet-config.toml:
+    exists: true
+    mode: "0644"
+
+command:
+  'run alumet-agent':
+    exec: "alumet-agent --plugins csv,procfs exec -- sleep 1 && cat alumet-output.csv"
```

Comment:
maybe use same plugins as the one generated as default config ? :
`csv,perf,procfs,socket-control`